skimage/metrics/setup.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from skimage._build import cython
+
+import os
+base_path = os.path.abspath(os.path.dirname(__file__))
+
+
+def configuration(parent_package='', top_path=None):
+    from numpy.distutils.misc_util import Configuration, get_numpy_include_dirs
+
+    config = Configuration('metrics', parent_package, top_path)
+    config.add_data_dir('tests')
+
+    cython(['_set_metrics.pyx'], working_path=base_path)
+
+    config.add_extension('_set_metrics', sources=['_set_metrics.c'],
+                         include_dirs=[get_numpy_include_dirs(), '../shared'])
+
+    return config
+
+
+if __name__ == '__main__':
+    from numpy.distutils.core import setup
+    setup(maintainer='scikit-image Developers',
+          maintainer_email='scikit-image@python.org',
+          description='Graph-based Image-processing Algorithms',
```

Comment:
Description should be changed, e.g. "Metrics to compare images"

Reply:
Done